### DISCUSSION TITLE
Is the wlr-output-management protocol implemented?

### DISCUSSION THREAD

**Discussion by wuyoli** [hyprwm/Hyprland#789] [Q&A] [answered]
Hi,
I wanted to install [kanshi](https://git.sr.ht/~emersion/kanshi), but saw, that kanshi requires the wlr-output-management protocol to be implemented. Is this the case with hyprland?

**Comment by vaxerski** [comment, marked as answer]
why dont u just try

yes kanshi works, but always prefer the config where possible. Altering the state externally sometimes leads to crashes.

Test it before asking questions about whether it will work :P

**Reply by wuyoli** [reply]
thanks

**Reply by calops** [reply]
If I may bounce back on this: personally, using kanshi *always* leads to a hyprland crash for me.
